coral/src/app/features/topics/details/history/TopicHistory.tsx

```diff
@@ -0,0 +1,83 @@
+import { DataTable, DataTableColumn, EmptyState } from "@aivenio/aquarium";
+import { useTopicDetails } from "src/app/features/topics/details/TopicDetails";
+import { TopicOverview } from "src/domain/topic";
+
+interface TopicHistoryRow {
+  id: number;
+  logs: string; //Remarks
+  team: string; //
```

Comment:
Empty comment? ^^

Reply:
🙈  I thought I removed all. Thanks for noticing.